tasks/cssmin.js

```diff
@@ -34,6 +34,11 @@ module.exports = function (grunt) {
 
       options.target = file.dest;
 
+      var unCompiledCssString = availableFiles.map(function (file) {
+        options.relativeTo = path.dirname(file);
```

Comment:
No point in doing this inside the iteration as it will only overwrite for each file. You can just set it outside using `file[0]` with the same effect.